src/style.css

```diff
@@ -0,0 +1,53 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
This is better to be set as 
```
* {
  box-sizing: border-box;
}
```
cause usually we want this behaviour for each element

Reply:
added